frontend/src/metabase-lib/lib/metadata/Database.js

```diff
@@ -41,6 +45,20 @@ export default class Database extends Base {
     return this.schemas.map(s => s.name).sort((a, b) => a.localeCompare(b));
   }
 
+  // TABLES
+
+  @memoize
+  tablesLookup() {
+    return createLookupByProperty(this.tables, "id");
+  }
+
+  // @deprecated: use tablesLookup
+  get tables_lookup() {
+    return this.tablesLookup();
+  }
```

Comment:
I like the approach of swapping out the properties for getters labeled as deprecated. Could we remove them entirely here?